code/modules/antagonists/blob/structures/factory.dm

```diff
@@ -44,3 +44,19 @@
 		BS.overmind = overmind
 		BS.update_icons()
 		overmind.blob_mobs.Add(BS)
+
+/obj/structure/blob/factory/lone //A blob factory that functions without a pulses
+
+/obj/structure/blob/factory/lone/Initialize(mapload, owner_overmind)
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/structure/blob/factory/lone/process()
+	addtimer(CALLBACK(src, /obj/structure/blob/factory/lone.proc/Be_Pulsed), 7 SECONDS)
```

Comment:
I don't understand what the point of having a 7 second callback here is.

Reply:
process is really fast, otherwise itll spawn all the spores REALLY fast, and constantly

Reply:
It still will. Did you test this? It will keep spawning them really fast, just 7 seconds later.

Reply:
Tested it, and it does seem to slow it down a good bit. Though there are already other cooldowns in.

Reply:
If you dont like this way, what do you suggest i were to do?

Reply:
It shouldnt now overlap.